README.md

```diff
@@ -1 +1,143 @@
-# anytv-kue
\ No newline at end of file
+# anytv-kue
+
+Kue Helper for setup and cleanup of Kue
+
+# Changes to be made to old codes
+===
+
+1. Replace all `require('kue');` with
+
+    ```javascript
+        const kue = require('anytv-kue')(config);
+    ```
+    
+# Added features
+===
+- Setup kue
+   
+    ```javascript
+    // server.js (before)
+    const kue = require('kue')
+
+    function start () {
+        /* server code */
+    }
+
+    queue.on('error', err => {
+        winston.log('error', 'QUEUE ERROR:', err);
+    });
+
+    process.once('SIGTERM', sig => {
+        winston.log('SIGTERM', sig);
+        queue.shutdown(5000, err => {
+            winston.log('error', 'Kue shutdown:', err );
+            process.exit(0);
+        });
+    });
+
+    queue.active((err, ids) => {
+        ids.forEach(id => {
+            kue.Job.get(id, (_err, job) => {
+                job.inactive();
+            });
+        });
+    });
+
+    queue.inactive((err, ids) => {
+        ids.forEach(id => {
+            kue.Job.get(id, (_err, job) => {
+                job.inactive();
+            });
+        });
+    });
+
+    start();
+    
+    ```
+    ```javascript
+    // server.js (now)
+    const kue = require('antv-kue')();
+
+    function start () {
+        /* server code */
+    }
+    
+    kue.setup();
```

Comment:
i think doing a load of jobs definition is not necessary for this library anymore because it will require the user to have a `config.JOBS_DIR` and a `.setup()` it will be easier for the user to do this manually in his code